lib/web/ui/usercontext.go

```diff
@@ -95,6 +95,8 @@ type userACL struct {
 	License access `json:"license"`
 	// Plugins defines whether the user has access to manage hosted plugin instances
 	Plugins access `json:"plugins"`
+	// Integrations defines whether the user has access to manage integrations.
+	Integrations access `json:"integrations"`
```

Comment:
@marcoandredinis i just realized that the frontend will need to check for both `integration.create and integration.use` before starting the aws integration b/c at the `list rds dbs` we will need to use the integration?

Reply:
Yes :+1: 
We'll need the `use` verb in order to call Integration's APIs

Reply:
Do we really need a new verb for this?

I'm concerned that we just add random verbs like this that are only used for one particular feature and then never document them, and after doing this for years we have a confusing RBAC system that no one can understand.

Is there a way to use the existing RBAC system that we have today rather than extend it just for thi feature?

Reply:
I don't think we have other existing verbs we can reuse here tbh. Other verbs we have are either for CRUD actions, or were also introduced specifically for individual features. For example, we have `VerbRotate`, `VerbCreateEnrolleToken`, `VerbEnroll`: https://github.com/gravitational/teleport/blob/master/api/types/constants.go. 

Compared to these, `VerbUse` is actually less "random" concept IMO, for example Kubernetes uses it (no pun intended) in its RBAC too.